internal/dto/env.go

```diff
@@ -2,23 +2,20 @@ package dto
 
 import (
 	"fmt"
-	"go.arcalot.io/log"
 	"os"
+
+	"go.arcalot.io/log"
 )
 
-func LookupEnvVar(key string, logger log.Logger) Verbose {
+// LookupEnvVar verifies that an environment variable is set and not empty
```

Comment:
I would consider this comment incomplete. It only describes part of it, which means it could confuse the user and hurt more than help.
It also needs a period at the end according to the linter.

Reply:
I have added more detail to the comments. Lmk if this is what you were looking for.